cmd/osbuild-composer/main.go

```diff
@@ -25,9 +24,15 @@ func main() {
 	flag.BoolVar(&verbose, "verbose", false, "Print access log")
 	flag.Parse()
 
-	var logger *log.Logger
-	if verbose {
-		logger = log.New(os.Stdout, "", 0)
+	verboseFound := false
+	flag.Visit(func(f *flag.Flag) {
```

Comment:
Why use `Visit` here? Since it's just a binary flag it's either true (specified) or false (not specified) no? And if specified display the info message.

Reply:
the flag being `false` means it's either default value or specified as false, no way to tell which, my initial idea was to issue the message if the flag is used (regardless of value)
But I think it's simpler to show the message when the behavior differs, so when verbose == false.